setup.cfg

```
@@ -23,6 +23,10 @@ license_file =  LICENSE
 [sdist]
 formats = bztar, zip
 
+[Pytest_Min_Version]
```

I think this section entry is wrong. According to the [pytest documentation](https://docs.pytest.org/en/6.2.x/customize.html?highlight=setup%20cfg#setup-cfg), the `setup.cfg` file needs to contain `tool:pytest`:

```suggestion
[tool:pytest]
```